styx-api-service/src/test/java/com/spotify/styx/api/WorkflowResourceTest.java

```diff
@@ -814,27 +815,39 @@ public void shouldFailInvalidWorkflowImage() throws Exception {
     assertThat(response, hasStatus(withCode(Status.BAD_REQUEST)));
   }
 
+
+
   @Test
   public void shouldReturnWorkflows() throws Exception {
     sinceVersion(Api.Version.V3);
-
-    Response<ByteString> response = awaitResponse(
-        serviceHelper.request("GET", path("")));
-
-    assertThat(response, hasStatus(withCode(Status.OK)));
-    assertJson(response, "[*]", hasSize(2));
+    var paths = List.of("", "?full", "?full=false");
```

Comment:
Please use parameterized test instead of a for loop.

Reply:
For this rather old code base, we have been using https://github.com/spotify/styx/blob/4a7175c1ae3d3715907e4bf63e99436dbe11e28c/styx-scheduler-service/src/test/java/com/spotify/styx/docker/LabelValueTest.java#L122-L126 instead.

Reply:
ok I couldn't make it work with parametrized test, thought it was due to the parent class already using parameters, but will try that

Reply:
it gives me:

```
org.junit.runners.model.InvalidTestClassError: Invalid test class 'com.spotify.styx.api.WorkflowResourceTest':
  1. Method shouldReturnWorkflows should have no parameters
  ```

Reply:
doesn't make sense to have multiple parameters anymore after changing full=true to full